tracer/src/Datadog.Trace/Configuration/TracerSettings.cs

```diff
@@ -229,6 +229,8 @@ public TracerSettings(IConfigurationSource source)
             IsDataStreamsMonitoringEnabled = source?.GetBool(ConfigurationKeys.DataStreamsMonitoring.Enabled) ??
                                         // default value
                                         false;
+
+            IsRareSamplerEnabled = source?.GetBool(ConfigurationKeys.RareSamplerEnabled) ?? false;
```

Comment:
When did we start adding `Is*` at the beginning of `*Enabled` settings? I see there are two others already, but should we keep doing it? I don't care much aside from the inconsistency.

Reply:
I don't know when we did it, but in .NET it's standard to have boolean properties start with `Is*`. So I'm probably to blame 😳